python/model.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+import argparse
+from lib.database import *
+from lib.tests import *
+from lib.colors import colors
+import random
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Database setup and testing script")
+    parser.add_argument(
+        "-q", "--quick", action="store_true", help="Perform quick setup"
+    )
+    parser.add_argument(
+        "-b",
+        "--batch-size",
+        type=int,
+        default=1e5,
+        help="Batch size for populating fact tables",
+    )
+    parser.add_argument(
+        "-m",
+        "--mode",
+        choices=["build", "test"],
+        default="build",
+        help="Mode: build or test",
+    )
+    parser.add_argument(
+        "-s",
+        "--step",
+        action="store_true",
+        help="Engage input() calls in the test routine for step-by-step execution",
+    )
+    args = parser.parse_args()
+
+    BE_QUICK_ABOUT_IT = args.quick
+    BATCH_SIZE = args.batch_size
+    BUILD = args.mode == "build"
+    TEST = args.mode == "test"
+    STEP = args.step
+
+    db = get_db_handle()
+
+    # fmt: off
+    if BUILD is True:
+        disconnect_other_users(db)
+        drop_schemata_except(db)
+        drop_public_entities(db)
+        ensure_extensions_exists(db)
+        pull_down_locations(db)
+        create_schemata(db)
+        create_lookup_tables(db)
+        artifically_descync_sequences_from_cris_data(db, ["cris_lookup.road_types_sequence", "cris_lookup.unit_types_sequence"])
```

Comment:
Because the IDs in the seed data start at 1, and the default value of a `serial primary key` `id`field also starts at 1, this function will jog the sequence up by 1K. This makes it more clear when you're looking at tabular data in the DB what is seed data and what is id/referential data in the DB.